### DISCUSSION TITLE
Restrict map to specific region

### DISCUSSION THREAD

**Discussion by adrianfilipeu** [FacilMap/facilmap#252] [Q&A]
Is it possible to restrict the collaborative map to a specific zoom-out level maybe and a specific region on the map (e.g. just one country/continent)?

**Comment by cdauth** [comment]
This would be rather easy to implement but probably difficult to provide a good UI where the settings can be configured. What would be your use case for this?

**Reply by adrianfilipeu** [reply]
Consider this use case (which is the one I am currently playing with): planning some trips with a group of friends. 
Using the collaborative map for all of us to pin their wish list locations. 
The only interest is e.g. Colombia. I can center the map but it would also be a lesser distraction and a bigger focus for not-so-techy users not being able to move away from the point-of-interest and even not to search places outside (I just thought of that while typing).

**Reply by adrianfilipeu** [reply]
Maybe just restrict map to view?
With an additional tick box in Manage Views?

![image](https://github.com/FacilMap/facilmap/assets/35315380/d2356d04-2398-47db-b1e2-32c93341a929)

Just spitting thoughts here haha

**Reply by adrianfilipeu** [reply]
Or somehow a separate section with "Restrictions". And only render the SVGs based on restriction inputs? I guess that could help with limiting search places as well
